daemon/api.go

```diff
@@ -634,21 +638,29 @@ func (inst *snapInstruction) Agreed(intro, license string) bool {
 var snappyInstall = snappy.Install
 
 func (inst *snapInstruction) install() interface{} {
+	// FIXME: handle license agreement
 	flags := snappy.DoInstallGC
 	if inst.LeaveOld {
 		flags = 0
 	}
-	_, err := snappyInstall(inst.pkg, inst.Channel, flags, inst)
-	if err != nil {
-		if inst.License != nil && snappy.IsLicenseNotAccepted(err) {
-			return inst.License
-		}
-		return err
-	}
+	state := inst.overlord.StateEngine().State()
+	state.Lock()
+	chg := state.NewChange("install-snap", fmt.Sprintf("Installing %s from %s", inst.pkg, inst.Channel))
```

Comment:
I suggest this as the format string:
```
format := "Install %q snap"
if inst.Channel != "stable" {
    format = "Install %q snap from %q channel"
}
```

In other words:
* No "ing" termination on summaries.. this is a requested change, not necessarily in progress
* %q rather than %s, so we don't have sentence being accidentally built
* The stable channel is the usual one.. no need to call it out